### PR TITLE
fix lost namespace in eval

### PR DESCRIPTION
**Pull request by fizzka**
Bugfix:
phpunit8 tearDown() declaration

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php [hunk 1]
```diff
@@ -34,9 +35,6 @@ protected function tearDown(): void
      */
     trait KernelShutdownOnTearDownTrait
     {
-        /**
-         * @return void
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php]: please keep the annotation

File: src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php [hunk 2]
```diff
@@ -16,7 +16,8 @@
 // Auto-adapt to PHPUnit 8 that added a `void` return-type to the tearDown method
 
 if (method_exists(\ReflectionMethod::class, 'hasReturnType') && (new \ReflectionMethod(TestCase::class, 'tearDown'))->hasReturnType()) {
-eval('
+    eval('
+    namespace Symfony\Bundle\FrameworkBundle\Test;
     /**
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php]: missing blank line :)

**Review by nicolas-grekas — COMMENTED** [review]
oups :)

File: src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php (hunk 2 above)
Comment (fizzka) [reply src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php]: fixed ;J

File: src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php (hunk 1 above)
Comment (fizzka) [reply src/Symfony/Bundle/FrameworkBundle/Test/KernelShutdownOnTearDownTrait.php]: reverted!

**Review by Tobion — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fizzka.

**Comment by nicolas-grekas** [comment]
added to 3.4 in da325fc0e2

**Pull request merged**